.travis.yml

```diff
@@ -4,9 +4,16 @@ jdk:
   - openjdk8
   - openjdk11
 
+env:
+  - TRAVIS_NODE_VERSION="10.9.0"
+    SCALAJS_VERSION="0.6.28"
+
 scala:
   - 2.13.0
 
+install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
```

Comment:
We don't have this in the other Scala modules' Travis-CI configs; do we need it here? What's it for?

Reply:
`nvm` is a nodejs version manager.
This is a technique for cross-building Scala.js against different Node.js versions.
Though this PR targets only `10.16.0` (current LTS).

Reply:
I see. I'm hesitant for two reasons:

* It looks like it will take time, and thus slow down PR validation.
* It looks like it could perhaps be fragile, and need maintenance.

Couldn't we just add this later, if and when we have a reason to build against multiple Node.js versions?

Reply:
OK, removed.